packages/ast-manipulation/js/imports.ts

```diff
@@ -35,7 +35,7 @@ export function addDefault(ast: AstTypes.Program, importFrom: string, importAs:
     addImportIfNecessary(ast, expectedImportDeclaration);
 }
 
-export function addNamed(ast: AstTypes.Program, importFrom: string, exportedAsImportAs: Record<string, string>) {
+export function addNamed(ast: AstTypes.Program, importFrom: string, exportedAsImportAs: Record<string, string>, isType = false) {
```

Comment:
I went with adding a new optional parameter to this existing function as the implementation of adding a type import vs a regular import were nearly identical.

If it's clearer to make a new function instead (e.g. `addNamedType`), I'm open to make that change as well

Reply:
I think that's a good decision